views/default/lightbox/elgg-colorbox-theme/colorbox.css

```diff
@@ -127,7 +123,7 @@
 }
 
 #cboxLoadingOverlay {
-	background: transparent url(graphics/ajax_loader_bw.gif) no-repeat center center;
+	background: transparent url(../../graphics/ajax_loader_bw.gif) no-repeat center center;
```

Comment:
It is. These get rewritten by our pre-processor, but even the relative link works fine.